pyomeca/frame_dependent.py

```diff
@@ -674,7 +674,7 @@ def to_numpy(self):
 
     # --- Plot method
 
-    def plot(self, x=None, ax=None, fmt="", lw=1, label=None, alpha=1):
+    def plot(self, *args, x=None, ax=None, **kwargs):
         """
         Plot a pyomeca vector3d (Markers3d, Analogs3d, etc.)
 
```

Comment:
We could explicitly say that *args and **kwargs are send to matplotlib's `plot` in the documentation